### DISCUSSION TITLE
Idea of how to create a dependency dropdown using Livewire3

### DISCUSSION THREAD

**Discussion by johnmambo** [livewire/livewire#7778] [1. Help]
I had a dependency dropdown that was working very well in Livewire 2, on upgrade to Livewire 3, it is not working.
How can I use
@php
//code
 @endphp
on a Livewire component?

**Comment by calebporzio** [comment]
You might need to add wire:key to the select. See this comment: https://github.com/livewire/livewire/pull/7500#issuecomment-1849108169

There is also docs in the main branch for this I believe and will be published soon.
